src/main/java/me/jellysquid/mods/sodium/client/render/chunk/ChunkRenderManager.java

```diff
@@ -467,6 +468,8 @@ public void updateChunks() {
         if (!futures.isEmpty()) {
             this.backend.upload(RenderDevice.INSTANCE.createCommandList(), new FutureDequeDrain<>(futures));
         }
+
+        this.builder.maybeIncreaseThreadLimit();
```

Comment:
`updateChunks` is used to essentially tick the chunk render manager. It's possible that other mods modify how this works or call into the builders directly, in which case putting this _here_ might cause performance limitations. I'm mainly thinking that this could cause issues with SeedQueue. But, based on preliminary testing, SeedQueue does seem to work fine - not confident on that though.